src/main/resources/modelling-fields/census/fraction-active-transport-to-work-data.json

```diff
@@ -0,0 +1,6 @@
+[
+  {
+    "importerClass": "uk.org.tombolo.importer.ons.CensusImporter",
+    "datasourceId": "qs701ew"
+  }
+]
```

Comment:
GitHub highlights files that dont end with an empty line

Reply:
ok. I will add those here.